client/src/elm/Translate.elm

```diff
@@ -593,7 +593,7 @@ type TranslationId
     | HealthEducationProvidedQuestion
     | HealthInsuranceQuestion
     | Heart
-    | HeartBpmUnitLabel
+    | BeatsPerMinuteUnitLabel
```

Comment:
@Diama1 
Note that we do our best to maintain alphabetcal order here, and when you rename something, you need to move it to proper place. I'll do this myself, so we can deploy the PR.